### ISSUE TITLE
Header managed in the same way as footer

### ISSUE THREAD

**Issue by pimarc** [coderedcorp/coderedcms#286]
Hello guys,

First of all I'm sorry for may bad english.

I'm just discovering this really awesome CMS. It's exactly what I need. It produces very simple html code output that I just need to style.
Congratulations for this work. I continue to investigate on it by trying a migration from an existing ecommerce site.

Now I have a suggestion. The footer is managed as a snippet. Why it's not the same for the header?
Sometimes we may have complex headers but you provide only a navigation snippet. What I'd like to have is a snippet header built in the same way than the footer and the possibility to integrate menus defined in the snipped navigations to this header as a streamfield. More generrally navigations should be available everywhere as a snippet like a card or carousel.

Maybe this can be done by using the reusable content feature but I'm not really sure about and in any case this is not intuitive.

Kind regards

Pierre

**Comment by vsalvino** [comment]
Yes, the navbar is a point of great contention. There are a few issues around it such as ability to support multisite. Wagtail itself has no navbar built in, so they have avoided the problem entirely :)

The recommendation for now, is that you will probably need to implement your own by overriding the `snippets/navbar.html` template as necessary.
